cmd/okctl/main.go

```diff
@@ -47,6 +47,10 @@ func buildRootCommand() *cobra.Command {
 	var outputFormat string
 
 	o := okctl.New()
+	if err := o.InitLogging(); err != nil {
+		fmt.Println("Error configuring logging: %w", err)
```

Comment:
As far as I know, only fmt.Errorf supports the %w thing(flag?)

Also concider using fmt.Fprintln() instead, with it you can pass output to stderr(os.Stderr)

Reply:
🙈 Fixed, thanks!